metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/BaseUserMetadataService.java

```diff
@@ -59,6 +60,15 @@ public void saveMetadata(final String userId, final HasMetadata holder, final bo
                 saveDataMetadata(dataDto.getDataUri(), userId, Optional.of(newMetadata), merge);
             }
         }
+
+        if (holder instanceof HasBusinessMetadata) {
+            final HasBusinessMetadata dataDto = ((HasBusinessMetadata) holder);
+            final ObjectNode newbusiMetadata = dataDto.getBusinessMetadata();
```

Comment:
`busi`? haha

Reply:
Not a good name, sounds like busy-Metadata. Will rename it.